src/Symfony/Component/Translation/Bridge/Phrase/Event/AbstractPhraseEvent.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation\Bridge\Phrase\Event;
+
+use Symfony\Component\Translation\TranslatorBag;
+use Symfony\Contracts\EventDispatcher\Event;
+
+/**
+ * @author wicliff <wicliff.wolda@gmail.com>
+ */
+abstract class AbstractPhraseEvent extends Event
```

Comment:
just a question of style, but i'd say the keyword `abstract` is enough

Reply:
another idea: what about renaming it to `AbstractProviderEvent` and move it to the `Symfony\Component\Translation\Provider` namespace?

Reply:
in hindsight, moving the abstract event to the provider namespace might not be the best of ideas as that will create an unnecessary dependency to the translation component. 

other than that naming the abstract class indeed is a matter of style / taste: personally i'd like to have a bit of clarity what a class is extending without having to go through the usages, but as it's an abstract class and thus won't be typehinted, i'm fine either way...